app/models/social_media_account_translation.rb

```diff
@@ -1,4 +1,12 @@
 class SocialMediaAccountTranslation < ApplicationRecord
+  belongs_to :social_media_account
+
   after_save :republish_organisation_to_publishing_api
   after_destroy :republish_organisation_to_publishing_api
+
+  def republish_organisation_to_publishing_api
```

Comment:
Is it worth adding a test for this method?

Reply:
A very similar method is included in another 7 models with no test :grimacing:.

We should probably look at DRYing this up and adding a test ... but for now the missing method is blocking a migration on deployment.

Reply:
Oh 🙈 

> We should probably look at DRYing this up and adding a test ... but for now the missing method is blocking a migration on deployment.

Sure thing, I'll give it a tick!